### DISCUSSION TITLE
Units of pole data in python/materials.py

### DISCUSSION THREAD

**Discussion by TimOfAllTrades** [NanoComp/meep#2524] [Q&A]
I had a question about some of the values in materials.py.

When using the Lorentz Drude model, I have an understanding that each pole had a proportionality constant * plasma frequency (usually merged into a single constant), a resonance frequency, and a damping constant.

The units I am normally used to working with are radians per second for each of those 3 values.

cSi_frq1 = 3.64 / um_scale
cSi_gam1 = 0
cSi_sig1 = 8
cSi_frq2 = 2.76 / um_scale
cSi_gam2 = 2 * 0.063 / um_scale
cSi_sig2 = 2.85
cSi_frq3 = 1.73 / um_scale
cSi_gam3 = 2 * 2.5 / um_scale
cSi_sig3 = -0.107

I see in the materials.py file, there is a frqN, gamN, and sigN with various values.  Do those 3 correspond to the quantities I mentioned earlier?  How do you convert the materials.py values into units of Rad/s?

Thank you and best regards

**Comment by smartalecH** [comment]
As [discussed in the docs](https://meep.readthedocs.io/en/latest/Materials/), meep internally multiplies by $2\pi$ where needed. So don't specify your coefficients in terms of radians. 

It's also not correct to universally use "seconds" either unless your length scale dictates that (see [discussion in docs](https://meep.readthedocs.io/en/latest/Introduction/#units-in-meep))

You can use the information above to map to an arbitrary unit.
